test/libsolidity/syntaxTests/parsing/mapping_with_names_struct_member_4.sol

```diff
@@ -0,0 +1,6 @@
+contract test {
+    struct Person {
+        mapping(uint phone => mapping(uint call => uint time) callTimes) friends;
```

Comment:
Adding another test case with an additional level of nesting would be nice, especially if it hits the conflicting parameter error between the root and leaf mapping, e.g.
```
mapping(uint phone => mapping(uint call => mapping(uint phone => something) leafMapping) callTimes) friends;
```

Reply:
There was a typo in the logic and this test case caught that. I've updated the fix in commit [7438248](https://github.com/ethereum/solidity/pull/13384/commits/743824807e95d2ca282996b50ea14f945a79c9f3).